static/panes/compiler.ts

```diff
@@ -466,6 +467,7 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         this.eventHub.unsubscribe();
         this.eventHub.emit('compilerClose', this.id, this.sourceTreeId ?? 0);
         this.editor.dispose();
+        this.compilerPicker.destroy();
```

Comment:
Fixing a bug while I was investigating it, the compiler picker wasn't being cleaned up after